tests/baselines/reference/genericConditionalConstrainedToUnknownNotAssignableToConcreteObject.errors.txt

```diff
@@ -39,8 +39,7 @@ genericConditionalConstrainedToUnknownNotAssignableToConcreteObject.ts(13,5): er
         M extends keyof T
     >(a2: ReturnType<T[M]>) {
         if (isA(a2)) {
-            // a2 is not narrowed
-            a2.x // error, but should be ok

```

Comment:
those were outdated comments, as we can see here - no errors are reported here (rightfully so). I took the liberty to fix those here